go.mod

```diff
@@ -255,7 +255,7 @@ require (
 
 replace github.com/minio/minio v0.0.0-20210206053228-97fe57bba92c => github.com/juicedata/minio v0.0.0-20240719032536-5d15c7c0135d
 
-replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20240425033113-7c40cb5eb3e9
+replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20240806064640-5dbad4fe9666
```

Comment:
can we rebase our patches onto upstream?

Reply:
Considering stability, only two relevant commits were cherry-picked, and there was no direct upgrade to version 2.2.

Reply:
We should follow the upstream as much as possible, and it's still early for 1.3 release, so we have enough time to merge with upstream, and fix the found issues.